tests/Dom_2/dom.js

```diff
@@ -0,0 +1,51 @@
+const file_path = process.env.FILE_PATH;
+const webdriver = require('selenium-webdriver');
+const assert = require('chai').assert;
+const { By } = require("selenium-webdriver");
+
+describe("Testing Scholars Project", function () {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder()
+        .forBrowser('chrome')
+        .build();
+
+    var actions = driver.actions();
+
+    before(function setupWebdriver(done) {
+        var file_url = `file://${file_path}`;
+        driver.get(file_url).then(done);
+    });
+
+    after(function () {
+        driver.quit();
+    });
+
+    it('test case: check dot on Cart', async () => {
+        const click = await driver.findElements(By.css("button"));
+        click[2].click();
+        const option = await driver.findElement(By.css("span")).isEnabled();
```

Comment:
I noticed that the other checks to find element has this `findElements` but variable option has `findElement`. Can you confirm that everything works fine